source/getting-started/implementation-plan.rst

```diff
@@ -87,11 +87,12 @@ A recommended installation of Mattermost Enterprise E20 configured with a redund
 
 This section provides a bibliography of key project references and deliverables that have been produced before this point in the project development.
 
-- Mattermost Product Overview: https://docs.mattermost.com/overview/index.html
-- Mattermost Software/Hardware Requirements: https://docs.mattermost.com/install/requirements.html 
-- Mattermost Security Features: https://docs.mattermost.com/overview/security.html 
-- Mattermost Administrator Guide: https://docs.mattermost.com/guides/administrator.html
-- Mattermost User Guide: https://docs.mattermost.com/guides/user.html
+- Mattermost Product Overview: https://docs.mattermost.com/about/product.html
+- Mattermost Software/Hardware Requirements: https://docs.mattermost.com/install/software-hardware-requirements.html 
+- Mattermost Security Features: https://docs.mattermost.com/about/security.html 
+- Mattermost Installation Guide: https://docs.mattermost.com/guides/install-deploy-upgrade-scale.html#install-mattermost
+- Mattermost Deployment Guide: https://docs.mattermost.com/guides/install-deploy-upgrade-scale.html#deploy-mattermost
+- Mattermost Upgrade Guide: https://docs.mattermost.com/guides/install-deploy-upgrade-scale.html#upgrade-mattermost
```

Comment:
The bottom 3 link to the same page (when clicking the links) cc @cwarnermm

Reply:
Thanks, @amyblais. These links were resolving to different pages, but new redirect added for another purpose is taking all three links to the same destination now. I'll improve this.